composer.json

```diff
@@ -15,14 +15,16 @@
         }
     },
     "require": {
-        "php": ">=7.0|>=8.0",
-        "doctrine/orm": "^2.0",
+        "php": "^7.1|^8.0",
+        "doctrine/orm": "^2.5",
         "psr/log": "^1.0|^2.0",
-        "symfony/property-access": "^2.8|^3.0|^4.0|^5.0",
-        "ext-json": "*"
+        "ext-json": "*",
+        "symfony/property-access": "^3.0|^4.0|^5.0|^6.0",
+        "doctrine/annotations": "^1.0|^2.0"
```

Comment:
Do we need this package in our code, or is it only necessary during test execution?